development/docker-compose.dev.yml

```diff
@@ -7,21 +7,13 @@ version: "3.8"
 services:
   nautobot:
     command: "nautobot-server runserver 0.0.0.0:8080"
-    healthcheck:
-      interval: "5s"
-      timeout: "5s"
-      start_period: "10m"  # intentionally conservative upper bound for `npm install` time in a fresh setup
-      retries: 3
-      test:
-        - "CMD"
-        - "curl"
-        - "-f"
-        - "http://localhost:8080/health/"
```

Comment:
This looks problematic, too, and should probably be fixed upstream

Reply:
Could you pls. better explain why is it problematic? Any proposal?

Reply:
Health check on local dev env is solely informational and causes more noise than worth IMHO